tests/globalhelper/helper.go

```diff
@@ -1,19 +1,25 @@
 package globalhelper
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"path"
 	"strings"
+	"time"
 
 	"github.com/golang/glog"
 	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	. "github.com/onsi/gomega"
```

Comment:
Please remove all gomega calls from helper package. Let's use ginkgo+gomega api only in test packages